Assets/Editor/UnitTests/Models/ScheduledEventTest.cs

```diff
@@ -17,6 +17,7 @@ public class ScheduledEventTest
 {
     private Action<ScheduledEvent> callback;
     private bool didRun = false;
+    private int runCount = 0;
```

Comment:
While you can get that the callback ran from checking the runCount, in most of the cases it seems (to me) that what is being looked for is simply that it did run, and it seems like the separate boolean makes it clearer what is being looked at. 

Of course, standard disclaimer that I'm far from an expert on tests of this sort, so I'm not sure exactly what is the preferred way to go about them.